src/proto.rs

```diff
@@ -110,3 +111,26 @@ impl TryInto<Content> for ContentProto {
         Content::from_proto(self.content, self.metadata.into()).map_err(Error::from)
     }
 }
+
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
Maybe I'm confused but I feel like we don't need this wrapper to serialize the group?